doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+* nRF21540 RF FEM - Range extender with integrated power amplifier (PA)/low-noise amplifier (LNA).
+* nPM1100 PMIC - Power Management IC with integrated dual-mode buck regulator and battery charger.
+
+For more information on the product, see the `Thingy:53 product page`_ and the `Nordic Thingy:53 User Guide`_.
+The |NCS| provides support for developing applications on the Thingy:53.
+
+.. _thingy53_serialports:
+
+Connecting to Thingy:53 serial ports
+************************************
+
+For connecting to Thingy:53, use a serial terminal.
+
+Thingy:53 uses USB transport and CDC ACM class.
+The CDC ACM baudrate is ignored, and transfer goes with USB speed.
+
+
+Firmware
+********
+
+Nordic Thingy:53 comes preprogrammed with the Edge Impulse firmware.
+This allows the Thingy:53 to connect to Edge Impulse Studio over Bluetooth LE using the nRF Edge Impulse mobile application, or through USB on your PC.
+The firmware allows you to capture sensor data, which in turn you can use to train and test machine learning models in Edge Impulse Studio.
+You can deploy trained machine learning models to the Nordic Thingy:53 through an over-the-air (OTA) update using Bluetooth LE, or update over USB using nRF Connect Programmer.
+
+See the `Nordic Semi Thingy:53 page`_ in the `Edge Impulse`_ documentation on how to use the Nordic Thingy:53 with the nRF Edge Impulse mobile application and Edge Impulse Studio.
+
+There are also multiple samples in the |NCS| that you can run on the Nordic Thingy:53.
+
+.. _thingy53_precompiled_fw:
+
+Programming precompiled firmware images
+***************************************
+
+Precompiled firmware image files are useful in the following scenarios:
+
+* Restoring the firmware to its initial image.
+* Updating the application firmware to a newly released version.
+* Testing different features of the Nordic Thingy:53.
+
+Downloading precompiled firmware images
+=======================================
+
+To obtain precompiled images for updating the firmware, perform the following steps:
+
+1. Go to the `Thingy:53 product page`_ and under the :guilabel:`Downloads` tab, navigate to :guilabel:`Precompiled applications`.
+#. Download and extract the latest Thingy:53 firmware package.
+#. Check the :file:`CONTENTS.txt` file in the extracted folder for the location and names of the different firmware images.
+
+Quick programming of precompiled firmware images
+================================================
+
+When you have the precompiled images ready, you can program the images directly onto the Thingy:53 using the nRF Connect Programmer applecation available in nRF Connect for Desktop.
+
+The Thingy:53 is connected directly to your PC through USB.
+This method uses the :doc:`mcuboot:index` feature and the built-in serial recovery mode of Thingy:53.
+
+You can also use the nRF Programmer mobile application for Android or iOS to update the firmware over-the-air (OTA) using Bluetooth Low Energy.
```

Comment:
This method assumes that application part of FW present on the Thingy53 does support it. All precompiled images support OTA using Bluetooth.

Reply:
Added mention of compatibility to OTA.